source/application/wintaser.cpp

```diff
@@ -390,7 +426,9 @@ static std::wstring TranslateDeviceName(const std::wstring& filename)
                         break;
                     }
                 }
-            } while (*p++);
+            }
+            while (*p++)
+                ;

```

Comment:
Other do-while statements didn't get formatted like this....

Reply:
Yeah, not sure what's up with this.